src/index.html

```diff
@@ -1,15 +1,11 @@
 <html>
   <head>
     <title>kenney.nl file browser</title>
```

Comment:
Based on changes in README this should probably be changed too

Reply:
@henrik-steen-adsk, are you pointing towards the `title`?

Reply:
Yes